crux-test/test/logback-test.xml

```diff
@@ -1,22 +1,46 @@
 <configuration>
-  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-    <encoder>
-      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
-    </encoder>
-  </appender>
-  <root level="WARN">
-    <appender-ref ref="STDOUT" />
-  </root>
-  <logger name="crux" level="INFO" />
-  <logger name="crux.node" level="WARN" />
-  <logger name="crux.standalone" level="ERROR" />
-  <logger name="crux.hash" level="WARN" />
-  <logger name="crux.http-server" level="WARN" />
-  <logger name="crux.kafka" level="WARN" />
-  <logger name="crux.jdbc" level="WARN" />
-  <logger name="crux.kv.memdb" level="ERROR" />
-  <logger name="crux.rdf" level="ERROR" />
-  <logger name="crux.tx" level="WARN" />
-  <logger name="org.apache.kafka" level="ERROR" />
-  <logger name="org.apache.zookeeper" level="ERROR" />
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <!-- use the previously created timestamp to create a uniquely
+        named log file -->
+        <file>logtester.log</file>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+        <filter class="ch.qos.logback.core.filter.EvaluatorFilter">
+            <evaluator> <!-- defaults to type ch.qos.logback.classic.boolex.JaninoEventEvaluator -->
+                <expression>logger.equals("crux.logging-test")</expression>
+            </evaluator>
+            <OnMismatch>DENY</OnMismatch>
+            <OnMatch>NEUTRAL</OnMatch>
+        </filter>
+    </appender>
+    <root level="trace">
+        <appender-ref ref="FILE" />
+    </root>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+            <level>WARN</level>
+        </filter>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <root level="WARN">
+        <appender-ref ref="STDOUT" />
+    </root>
+
+    <logger name="crux" level="trace" />
```

Comment:
these loggers form a tree - i.e. `"crux"` is a parent logger of `"crux.node"` - unless you override it, `crux.node` will take on the level of `crux` (and they'll both take on the root logger level if neither is provided)